tests/queries/0_stateless/02232_non_negative_derivative.sql

```diff
@@ -0,0 +1,43 @@
+DROP TABLE IF EXISTS nnd;
+
+CREATE TABLE nnd
+(
+    id Int8, ts DateTime64(3, 'UTC'), metric Float64
+)
+ENGINE=MergeTree()
+ORDER BY id
+SETTINGS index_granularity = 8192;
+
+truncate table nnd;
+system stop merges nnd;
```

Comment:
Any specific reason why there must be no merges?

Reply:
No specific reason, but I want to be sure table is not changed during the test